src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
+        function result = subtract(a, b)
+            result = a - b;
+        end
+        
+        function result = multiply(a, b)
+            result = a * b;
+        end
+        
+        function result = divide(a, b)
+            if b == 0
+                error('Division by zero is undefined.');
+            end
+            result = a / b;
```

Comment:
Using 'error' for handling division by zero is acceptable, but consider using exceptions for better error handling.